version_specific_results/devel_performance_310_requirements.txt

```diff
@@ -15,7 +15,7 @@ numpy==1.23.5
 packaging==22.0
 Pillow==9.3.0
 ply==3.11
-pyccel @ file:///home/runner/work/pyccel-benchmarks/pyccel-benchmarks/pyccel
+pyccel @ git+https://github.com/pyccel/pyccel@4e703de8b2a95eac64fb8971269d987c5ba36f49
 pyccel-benchmarks @ file:///home/runner/work/pyccel-benchmarks/pyccel-benchmarks
```

Comment:
Just checking - should this one be updated too?

Reply:
Absolutely! 😅

Reply:
I'm not sure why this only appears in this file though!

Reply:
I'll take care of this